### ISSUE TITLE
Setting start time >1 on video effects where xLights has "Hardware Video Decoding" Enabled causes the effect to display a blue screen.

### ISSUE THREAD

**Issue by bjhiltbrand** [smeighan/xLights#3829]
**Describe the bug**
Either importing or creating a new sequence, if you have xLights preferences set to enable "Hardware Video Decoding", and if you work with a video effect and set the "Start Time" value to anything other than 0, 1, or 7(oddly enough), The video effect will only show a blue screen. I noticed this first when importing Visionary Light Show's "Master of Puppets" sequence. After working with Robert Gray and Kyle Bostick, we found that Robert could not reproduce the problem on his PC, and Kyle could not reproduce it on his Mac. Then, I unchecked "Hardware Video Decoding" on my PC and re-rendered, and it worked for me. Kyle verified that the issue was reproducible on his PC, but not his Mac. Therefore, it looks like it only affects PC installations. Kyle mentioned that he had hardware decoding on his PC enabled last Halloween when he used the sequence last, and it worked then. So we think it was a bug introduced somewhere between Halloween and now.

**To Reproduce**
Links:
Original question in Facebook:
https://www.facebook.com/groups/628061113896314/?multi_permalinks=6342861952416173&ref=share
Screen capture reproducing bug:
https://nas.hiltbrandhome.com/d/s/tpzhgsMYwj69YBl68VUxSjUUuYKpl2SW/ma9wKv0L8TnHMmnjs1dY9MZg5qMVlZPk-prXgNjJ4fQo
Please let me know where to send the sequence if necessary, as I don't want to leave it available in this bug report (It's a paid sequence from a vendor)

1. Ensure that Hardware Video Decoding is enabled in xLights Preferences
2. Import an existing sequence or create a new one
3. Select a video effect or add a new one.
4. Set video start time to any number greater than 1
5. See the model that you are working with go blue

**Expected behavior**
Video should play at the specified start time on the model.

**Screenshots**
https://nas.hiltbrandhome.com/d/s/tpzhgsMYwj69YBl68VUxSjUUuYKpl2SW/ma9wKv0L8TnHMmnjs1dY9MZg5qMVlZPk-prXgNjJ4fQo

**Versions (please complete the following information):**
 - OS: Windows 10 and Windows 11 - Affected; Mac (Not sure of the version) - Not Affected
 - xLights version Tested on 2023.01, 2023.02, 2023.06, 2023.07, and 2023.08 - All Affected

**Comment by bjhiltbrand** [comment]
Just tested this on 2023.10. Still an issue.

**Comment by keithsw1111** [comment]
I tried it on my laptop with intel video built in and it seems to work ok. Maybe you can attach your logs after a render and there may be something in there.

**Comment by bjhiltbrand** [comment]
[xlights_log_07-04-23_bjhiltbrand.log](https://github.com/smeighan/xLights/files/11953200/xlights_log_07-04-23_bjhiltbrand.log)

**Comment by bjhiltbrand** [comment]
Looks like there are a bunch of errors that look like this:

2023-07-04 20:00:33,433 29764 log_base [ERROR] WHVD: Seek failed.

**Comment by cybercop23** [comment]
@bjhiltbrand there was a recent fix in 2024.2 for video seek. Can you please test it and/or if no longer an isse close this?

**Comment by bjhiltbrand** [comment]
> @bjhiltbrand there was a recent fix in 2024.2 for video seek. Can you please test it and/or if no longer an issue close this?

Nope, no go. It still causes the blue screen if Hardware Video Decoding is turned on. The "[ERROR] WHVD: Seek failed." messages are still in the logs as well.

**Comment by coolguymatt** [comment]
I have this issue as well.

**Comment by bjhiltbrand** [comment]
@cybercop23 This looks like it has been partially fixed in the latest release (Possibly 2024.6, 2024.7, or 2024.8). The trouble sequence is now rendering and playing fine when using Hardware Decoding and setting it to FFMPEG CUDA (I have a machine with an RTX 3070 and a couple with 1050Ti video cards). I can actually set it to any of the FFMPEG settings and it will work fine now. Selecting DirectX 11 still causes it to fail.

**Comment by cybercop23** [comment]
So why just patrially fixed? What else is not working?

**Comment by bjhiltbrand** [comment]
The fact that selecting DirectX 11 still results in a blue screen when a video is rendered. If that is a known issue and should not keep this issue from being closed out, let me know and I will gladly close the issue.

**Comment by bjhiltbrand** [comment]
I totally forgot about this bug report. I just followed my video that I had uploaded to reproduce it, and it still does the same thing when DirectX11 is selected and a video is used with a start time >1. Here is the relevant log:

`2026-02-16 11:15:00,739 11464 log_base [DEBUG] WHVD: Hardware Video Decoder Initialised OK for video: X:\xLights\Resources\Videos\Metallica - Master of Puppets\stranger things master of puppets.mp4
2026-02-16 11:15:00,739 11464 log_base [INFO] Video loaded: X:\xLights\Resources\Videos\Metallica - Master of Puppets\stranger things master of puppets.mp4
2026-02-16 11:15:00,739 11464 log_base [INFO]       Length MS: 224347.00
2026-02-16 11:15:00,739 11464 log_base [INFO]       _frames: 5471
2026-02-16 11:15:00,739 11464 log_base [INFO]       Frames per second 24.39
2026-02-16 11:15:00,739 11464 log_base [INFO]       Source size: 1280x720
2026-02-16 11:15:00,739 11464 log_base [INFO]       Output size: 38x35
2026-02-16 11:15:00,739 11464 log_base [INFO]       Alpha: TRUE
2026-02-16 11:15:00,739 11464 log_base [INFO]       Frame ms 41
2026-02-16 11:15:00,777 11464 log_base [DEBUG] Video effect initialising ... seeking to start location for the video 69.300003.
2026-02-16 11:15:00,787 11464 log_base [ERROR] WHVD: Seek failed.
2026-02-16 11:15:00,787 11464 log_base [DEBUG] Video effect length: 2150, video length: 224347, startoffset: 69.300003, duration treatment: Normal.
2026-02-16 11:15:00,792 11464 log_base [ERROR] WHVD: Seek failed.
2026-02-16 11:15:00,797 11464 log_base [ERROR] WHVD: Seek failed.
2026-02-16 11:15:00,801 11464 log_base [ERROR] WHVD: Seek failed.`

FFMPEG CUDA still works fine.

`When you say "results in a blue screen" is that a blue screen crash like the whole computer is locked up or just the model is showing blue?`

The model shows blue for the duration of the video effect.

**Comment by derwin12** [comment]
I wonder if its the video then .. I tried on mine and didnt have an issue.

**Comment by derwin12** [comment]
Can you send me the video? darylerwin at gmail
